tests/test_FFBbasis3D.py

```diff
@@ -498,3 +498,23 @@ def testFFBBasis3DExpand(self):
         ]
 
         self.assertTrue(np.allclose(result, ref, atol=1e-2))
+
```

Comment:
For this and the remaining PSWF FPSWF see remark for FB3D.

Reply:
As discussed, added `pswf_params_2d` and implemented similar change as for the 3D